packages/@lwc/template-compiler/src/__tests__/parser.spec.ts

```diff
@@ -416,6 +416,68 @@ describe('expression', () => {
         expect(result.warnings).toHaveLength(0);
     });
 
+    it('forbids extra curly brackets on text nodes', () => {
+        // Note: the special case with curly brackets on text nodes is because of Text node traversal does not properly
+        // builds the expression: const tokenizedContent = rawText.split(EXPRESSION_RE);
```

Comment:
How the expression is parsed is an internal detail. If we change the code, this comment will be outdated.
Let's remove the comment from here.